pyproject.toml

```diff
@@ -54,6 +54,8 @@ cohere = ["cohere>=5.11.2,<6.0.0"]
 colpali-engine = ["colpali-engine>=0.3.10,<0.4.0"]
 dse = ["qwen-vl-utils==0.0.8"]
 jina-clip = ["timm>=1.0.7,<2.0.0"]
```

Comment:
Can you also pin the transformers version here for the jcv2 model? I think it breaks with any version above 4.52.0